src/main/java/seedu/clinic/logic/commands/DeleteCommand.java

```diff
@@ -33,7 +33,10 @@
 public class DeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "delete";
-
+    public static final String COMPLETE_DELETE_SUPPLIER_COMMAND = "delete ct/s i/";
+    public static final String COMPLETE_DELETE_WAREHOUSE_COMMAND = "delete ct/w i/";
+    public static final String COMPLETE_DELETE_SUPPLIER_PRODUCT_COMMAND = "delete ct/s i/ pd/";
```

Comment:
Oops the ct for SUPPLIER_PRODUCT and WAREHOUSE_PRODUCT should be ps and pw.

Reply:
Oh yes I'll fix this, good catch!